templates/personal.php

```diff
@@ -20,10 +20,10 @@
 				<button id="zoteroAPIKeySave"><span title="<?php p($l->t('Save Zotero API key')); ?>" data-toggle="tooltip">Save</span></button>
 				<button id="zoteroAPIKeyRemove"><span  class="icon-delete" title="<?php p($l->t('Remove Zotero API Key')); ?>" data-toggle="tooltip"></span></button>
 			</p>
-			<p><em><?php p($l->t('To use Zotero specify your API key here. You can create your API key in your ')); ?> <a href="https://www.zotero.org/settings/keys" target="_blank"><?php p($l->t('Zotero account API settings.')); ?></a></em></p>
+			<p><em><?php p($l->t('To use Zotero specify your API key here. You can create your API key in your') + ' '); ?> <a href="https://www.zotero.org/settings/keys" target="_blank"><?php p($l->t('Zotero account API settings.')); ?></a></em></p>
```

Comment:
```suggestion
			<p><em><?php p($l->t('To use Zotero specify your API key here. You can create your API key in your')); ?> <a href="https://www.zotero.org/settings/keys" target="_blank"><?php p($l->t('Zotero account API settings.')); ?></a></em></p>
```

Reply:
`+` is not a valid operator on strings in PHP but the space would already be there before the `<a>` tag